ProcessLib/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -245,6 +245,14 @@ void HydroMechanicsProcess<DisplacementDim>::initializeConcreteProcess(
         MeshLib::getOrCreateMeshProperty<double>(
             const_cast<MeshLib::Mesh&>(mesh), "pressure_interpolated",
             MeshLib::MeshItemType::Node, 1);
+
+    // Initialize local assemblers after all variables have been set.
+    const int process_id = 0;
```

Comment:
Suggestion:
- Remove lines 251-251.
- Change `GlobalExecutor::executeSelectedMemberOnDereferenced` to `GlobalExecutor::executeMemberOnDereferenced`
as 
```
  GlobalExecutor::executeSelectedMemberOnDereferenced(
        &LocalAssemblerInterface::initialize, _local_assemblers,
         *_local_to_global_index_map);
```

The same for other processes.

Reply:
using `GlobalExecutor::executeMemberOnDereferenced`

Reply:
Sorry: Remove lines 250-251.

Reply:
Thanks! This is better. Rebased. Changes in last commit.